recipes/polyglotdb/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "polyglotdb" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 89d2b4a19595a7bc14d1fba124afc63454baa9678275cd093d091d0430fc36c2
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - conch_sounds
+    - influxdb
+    - librosa
+    - neo4j-python-driver
+    - pip
+    - praatio
+    - python
+    - requests
+    - textgrid
+    - tqdm
```

Comment:
Are you sure all of this is needed?

Reply:
These are direct dependencies, I don't think the package would build/run without them, unless I'm missing something.

Reply:
`Host` are packages that are needed when building the package: `pip`, `cmake` etc. `Run` is what is used when the package is installed.

Ref: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-host-and-run